### PR TITLE
bugfix: global session is not change to Committed in saga mode

### PR DESCRIPTION
**Pull request by wingchi-leung**
- [x] I have registered the PR [changes](https://github.com/seata/seata/tree/develop/changes).

### Ⅱ. Does this pull request fix one issue?

fixes https://github.com/seata/seata/issues/5045

### CODE REVIEW

**Review by a364176773 — APPROVED** [review]
LGTM @slievrly PTAL

File: changes/en-us/develop.md [hunk 1]
```diff
@@ -29,7 +29,9 @@ Add changes here for all PR submitted to the develop branch.
 - [[#5018](https://github.com/seata/seata/pull/5018)] fix loader path in startup scripts
 - [[#5004](https://github.com/seata/seata/pull/5004)] fix duplicate image row for update join
 - [[#5033](https://github.com/seata/seata/pull/5033)] fix null exception when sql columns is empty for insert on duplicate
-- [[#5045](https://github.com/seata/seata/pull/5045)] fix global session is not change to Committed
+- [[#5038](https://github.com/seata/seata/pull/5038)] remove @EnableConfigurationProperties({SagaAsyncThreadPoolProperties.class})
+- [[#5050](https://github.com/seata/seata/pull/5045)] fix global session is not change to Committed
```
Comment (a364176773) [inline changes/en-us/develop.md:33]: #5045

**Comment by tuwenlin** [comment]
LGTM

**Review by tuwenlin — APPROVED** [review]
LGTM

**Comment by Bughue** [comment]
LGTM

**Review by jsbxyyx — APPROVED** [review]
LGTM

File: changes/zh-cn/develop.md [hunk 2]
```diff
@@ -32,6 +32,7 @@
 - [[#5004](https://github.com/seata/seata/pull/5004)] 修复mysql update join行数据重复的问题
 - [[#5033](https://github.com/seata/seata/pull/5033)] 修复InsertOnDuplicateUpdate的SQL语句中无插入列字段导致的空指针问题
 - [[#5038](https://github.com/seata/seata/pull/5038)] 修复SagaAsyncThreadPoolProperties冲突问题
+- [[#5050](https://github.com/seata/seata/pull/5050)] 修复Saga模式下全局状态未正确更改成Commited
```
Comment (caohdgege) [inline changes/zh-cn/develop.md:35]: fix一下typo，`Committed`

File: changes/en-us/develop.md [hunk 3]
```diff
@@ -30,6 +30,7 @@ Add changes here for all PR submitted to the develop branch.
 - [[#5004](https://github.com/seata/seata/pull/5004)] fix duplicate image row for update join
 - [[#5033](https://github.com/seata/seata/pull/5033)] fix null exception when sql columns is empty for insert on duplicate
 - [[#5038](https://github.com/seata/seata/pull/5038)] remove @EnableConfigurationProperties({SagaAsyncThreadPoolProperties.class})
+- [[#5050](https://github.com/seata/seata/pull/5050)] fix global session is not change to Committed
```
Comment (caohdgege) [inline changes/en-us/develop.md:33]: 英文版md和标题提现一下saga

**Review by caohdgege — APPROVED** [review]
LGTM

**Review by slievrly — COMMENTED** [review]
LGTM
There is a follow-up issue that needs attention, `success` is always true in AT transaction mode.

**Pull request merged**